test/files/stream.iced

```diff
@@ -0,0 +1,120 @@
+stream = require('../../src/stream.iced')
+enc = require('../../src/encoding.iced')
+b58enc = enc.b58
+b62enc = enc.b62
+b64enc = enc.b64
+
+b64stripped =
+	encode : (x) ->
+		r = new RegExp('=', 'g')
+		x.toString('base64').replace(r, '')
+	decode : (x) ->
+		until (x.length % 4) is 0
+			x += "="
+		return new Buffer x, 'base64'
+
+#==========================================================
+#These tests encode then immediately decode, and compare the result to the original text
+#=========================================================
+
+exports.test_b58_consistency = (T, cb) ->
+
+	for i in [1...1000]
```

Comment:
make sure your test data goes over the block boundary, i don't think you'll get there if you do this.